crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -33,9 +33,9 @@ pub enum TransactionType {
 }
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
-struct MerchantAuthentication {
-    name: String,
-    transaction_key: String,
+pub struct MerchantAuthentication {
+    pub name: String,
+    pub transaction_key: String,
```

Comment:
The struct fields need not be pub right? it's sufficient if the struct is pub I think